include/avif/avif.h

```diff
@@ -402,6 +402,8 @@ typedef struct avifRGBImage
 
     uint8_t * pixels;
     uint32_t rowBytes;
+    avifBool ignoreAlpha;                  // Used for XRGB formats, treats formats containing alpha (such as ARGB) as if they were
```

Comment:
Can you move these two new lines just below `chromaUpsampling`, and move the associated `ignoreAlpha` initialization in `avifRGBImageSetDefaults()`? I'll accept the PR after that.

Reply:
Done, thanks!